src/api.js

```diff
@@ -671,18 +673,44 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
     * We query it like that:
     * ```javascript
     * var db = new SQL.Database();
-    * var res = db.exec("SELECT id FROM test; SELECT age,name FROM test;");
+    * var res = db.exec(
+    *     (
+    *         "DROP TABLE IF EXISTS test;\n"
+    *         + "CREATE TABLE test (id INTEGER, age INTEGER, name TEXT);\n"
+    *         + "INSERT INTO test VALUES ($id1, :age1, @name1);\n"
+    *         + "INSERT INTO test VALUES ($id2, :age2, @name2);\n"
+    *         + "INSERT INTO test VALUES ($id3, :age3, @name3);\n"
+    *         + "SELECT id FROM test;\n"
+    *         + "SELECT age,name FROM test;\n"
+    *     ),
+    *     {
+    *         "$id1": 1,
+    *         ":age1": 1,
+    *         "@name1": "Ling",
+    *         "$id2": 2,
+    *         ":age2": 18,
+    *         "@name2": "Paul",
+    *         "$id3": 3,
+    *         ":age3": 3,
+    *         "@name3": "Marcus"
+    *     }
+    * );
```

Comment:
```suggestion
    * var res = db.exec(
    *            "CREATE TABLE test (id INTEGER, age INTEGER, name TEXT);"
    *         + "INSERT INTO test VALUES ($id1, :age1, @name1); "
    *         + "SELECT age,name FROM test WHERE id=$id1",
    *         { "$id1": 1, ":age1": 1, "@name1": "Ling",  }
    * );
```

Reply:
the documented example does not produce the documented result.
if intent is to have a shorter, nicer looking example, then i compromised with following in commit 6e5024ed6c522e2719bb37e9418f708e29cf3786 if its ok:

```diff
     * multi-line statement using params:
     *
     * | id | age |  name  |
     * |:--:|:---:|:------:|
     * | 1  |  1  | Ling   |
     * | 2  |  18 | Paul   |
-    * | 3  |  3  | Markus |
     *
     * We query it like that:
     * ```javascript
     * var db = new SQL.Database();
     * var res = db.exec(
-    *     (
-    *         "DROP TABLE IF EXISTS test;\n"
-    *         + "CREATE TABLE test (id INTEGER, age INTEGER, name TEXT);\n"
-    *         + "INSERT INTO test VALUES ($id1, :age1, @name1);\n"
-    *         + "INSERT INTO test VALUES ($id2, :age2, @name2);\n"
-    *         + "INSERT INTO test VALUES ($id3, :age3, @name3);\n"
-    *         + "SELECT id FROM test;\n"
-    *         + "SELECT age,name FROM test;\n"
-    *     ),
+    *     "DROP TABLE IF EXISTS test;\n"
+    *     + "CREATE TABLE test (id INTEGER, age INTEGER, name TEXT);"
+    *     + "INSERT INTO test VALUES ($id1, :age1, @name1);"
+    *     + "INSERT INTO test VALUES ($id2, :age2, @name2);"
+    *     + "SELECT id FROM test;"
+    *     + "SELECT age,name FROM test WHERE id=$id1",
     *     {
-    *         "$id1": 1,
-    *         ":age1": 1,
-    *         "@name1": "Ling",
-    *         "$id2": 2,
-    *         ":age2": 18,
-    *         "@name2": "Paul",
-    *         "$id3": 3,
-    *         ":age3": 3,
-    *         "@name3": "Marcus"
+    *         "$id1": 1, ":age1": 1, "@name1": "Ling",
+    *         "$id2": 2, ":age2": 18, "@name2": "Paul"
     *     }
     * );
     * ```
@@ -700,8 +689,8 @@ Module["onRuntimeInitialized"] = function onRuntimeInitialized() {
     * `res` is now :
     * ```javascript
     *     [
-    *         {"columns":["id"],"values":[[1],[2],[3]]},
-    *         {"columns":["age","name"],"values":[[1,"Ling"],[18,"Paul"],[3,"Marcus"]]}
+    *         {"columns":["id"],"values":[[1],[2]]},
+    *         {"columns":["age","name"],"values":[[1,"Ling"]]}
     *     ]
     * ```
```

![image](https://user-images.githubusercontent.com/280571/76126101-dbda4e00-5fc3-11ea-878c-68d5441b64f3.png)

Reply:
This seems reasonable to me 👍